package.json

```diff
@@ -2,9 +2,10 @@
     "name": "playcanvas-react-monorepo",
     "description": "A monorepo for PlayCanvas React renderer and examples. Please see the packages directory for more information.",
     "private": true,
+    "license": "MIT",
```

Comment:
Presumably you don't want the private field any more?

Reply:
It's on the root monorepo package.json, which should be private, the library package.json is public thoug